aerich/migrate.py

```diff
@@ -50,7 +50,7 @@ class Migrate:
     @classmethod
     def get_all_version_files(cls) -> List[str]:
         return sorted(
-            filter(lambda x: x.endswith("py"), os.listdir(cls.migrate_location)),
+            filter(lambda x: x.endswith("py") and not x.startswith("__"), os.listdir(cls.migrate_location)),
```

Comment:
Instead of this approach. Why dont we do `x.startswith('__init__)'`.

Reply:
Thanks for your review, I have edited it, please check again.